### ISSUE TITLE
Have sprite give more descriptive href's

### ISSUE THREAD

**Issue by amyrlam** [hashicorp/flight#272]
IS:

```
<symbol id="activity-16" viewBox="0 0 16 16"><path fill-rule="evenodd" d="M6.016 1a.75.75 0 01.698.521l3.306 10.33 1.47-4.341A.75.75 0 0112.2 7H15a.75.75 0 010 1.5h-2.262l-2.028 5.99a.75.75 0 01-1.424-.011L5.952 4.06 4.504 8.008A.75.75 0 013.8 8.5H1A.75.75 0 111 7h2.276l2.02-5.508c.11-.301.4-.499.72-.492z" clip-rule="evenodd"/></symbol>
```

and 

![image](https://user-images.githubusercontent.com/1372946/134598180-55ea0a27-e71f-46c8-acc5-fbb1d161ab8a.png)

SHOULD BE:

```
<symbol id="flight-activity-16" viewBox="0 0 16 16"><path fill-rule="evenodd" d="M6.016 1a.75.75 0 01.698.521l3.306 10.33 1.47-4.341A.75.75 0 0112.2 7H15a.75.75 0 010 1.5h-2.262l-2.028 5.99a.75.75 0 01-1.424-.011L5.952 4.06 4.504 8.008A.75.75 0 013.8 8.5H1A.75.75 0 111 7h2.276l2.02-5.508c.11-.301.4-.499.72-.492z" clip-rule="evenodd"/></symbol>
```

Proposal:
- Have a more specific namespaced id in sprite generation
  - So that the href won't collide, in the off chance there is same name
  - Arguably more clear to the dev where it's coming from if was `href=#flight-activity-16` or some other name

Came up in pairing with @DingoEatingFuzz so wanted to float the idea

**Comment by didoo** [comment]
@amyrlam great catch, yeah you're absolutely right. It will be fixed in #271.

**Comment by amyrlam** [comment]
awesome thanks!!
